src/components/Chat/ChatMessage.tsx

```diff
@@ -100,56 +90,21 @@ class ChatMessage extends UIComponent<Extendable<ChatMessageProps>, any> {
         {...rest}
         className={className}
       >
-        {childrenPropExists ? children : this.renderContent(classes, styles, variables)}
+        {childrenPropExists ? (
+          children
+        ) : (
+          <>
+            {!mine &&
+              Text.create(author, { defaultProps: { size: 'small', styles: styles.author } })}
```

Comment:
Here the mine prop in my opinion has the right usage (don't show the author if it is me). I am not 100% sure about this though, I want to hear your opinion on that.

Reply:
it was like this even before, it's just refactored now